File: app/src/main/java/no/nav/sosialhjelp/soknad/consumer/exceptions/PdlApiException.java
package no.nav.sosialhjelp.soknad.consumer.exceptions;

import no.nav.sosialhjelp.soknad.domain.model.exception.SosialhjelpSoknadApiException;

public class PdlApiException extends SosialhjelpSoknadApiException {

    public PdlApiException(String message) {
        super(message);
    }

    public PdlApiException(String message, Throwable t) {
        super(message, t);
    }
}

File: src/main/kotlin/no/nav/sosialhjelp/soknad/client/exceptions/PdlApiException.kt
package no.nav.sosialhjelp.soknad.client.exceptions

import no.nav.sosialhjelp.soknad.common.exceptions.SosialhjelpSoknadApiException

class PdlApiException : SosialhjelpSoknadApiException {
    constructor(message: String?) : super(message) {}
    constructor(message: String?, t: Throwable?) : super(message, t) {}
}
